src/vs/workbench/contrib/terminalContrib/accessibility/browser/textAreaSyncAddon.ts

```diff
@@ -65,10 +65,10 @@ export class TextAreaSyncAddon extends Disposable implements ITerminalAddon {
 
 		this._updateCommandAndCursor();
 
-		if (this._currentCommand !== textArea.value) {
+		if (this._currentCommand) {
```

Comment:
this check was causing issues

Reply:
actually, just tested again and saw the same issue with this check removed, so reverting that.